aten/src/ATen/native/Convolution.cpp

```diff
@@ -529,7 +529,8 @@ struct ConvParams {
            !transposed &&   // or transposed tensors
            input.ndimension() == 4 && // must be in NCHW format
            weight.ndimension() == 4 &&
-           (at::symint::size<T>(weight, 2) < 17) && (at::symint::size<T>(weight, 3) < 17) // NNPACK only supports kernels up to 16x16
+           (at::symint::size<T>(weight, 2) < 17) && (at::symint::size<T>(weight, 3) < 17) && // NNPACK only supports kernels up to 16x16
+           (padding[0] >= at::symint::size<T>(weight, 2)) && (padding[1] >= at::symint::size<T>(weight, 3))
```

Comment:
```suggestion
           (padding[0] < at::symint::size<T>(weight, 2)) && (padding[1] < at::symint::size<T>(weight, 3))
```
Also, add a comment to explain the check here?

Reply:
Add a comment.